stable/spring-cloud-data-flow/values.yaml

```diff
@@ -40,6 +42,8 @@ skipper:
   platformName: default
   service:
     type: ClusterIP
+  # If not set, use a default configMap else use the configMap from specified value
```

Comment:
we can remove this comment as its already defined in the readme

Reply:
Removed